sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/MssqlDatabaseIT.java

```diff
@@ -5,22 +5,30 @@
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.bootstrap.SqlServerService;
 import io.quarkus.test.scenarios.QuarkusScenario;
-import io.quarkus.test.scenarios.annotations.DisabledOnFipsAndJava17;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.services.Dependency;
 import io.quarkus.test.services.QuarkusApplication;
 import io.quarkus.test.services.SqlServerContainer;
 
-@DisabledOnFipsAndJava17(reason = "https://github.com/quarkusio/quarkus/issues/40813")
+@DisabledOnNative(reason = "BouncyCastle JSSE FIPS doesn't work in native but is required for FIPS-enabled environments")
 @DisabledIfSystemProperty(named = "ts.arm.missing.services.excludes", matches = "true", disabledReason = "https://github.com/quarkus-qe/quarkus-test-suite/issues/2017")
 @QuarkusScenario
 public class MssqlDatabaseIT extends AbstractSqlDatabaseIT {
 
     @SqlServerContainer(tlsEnabled = true)
     static SqlServerService database = new SqlServerService();
 
-    @QuarkusApplication
+    @QuarkusApplication(dependencies = {
+            // added here as BouncyCastle JSSE FIPS is not compatible with native mode
+            @Dependency(groupId = "io.quarkus", artifactId = "quarkus-security"),
+            @Dependency(groupId = "org.bouncycastle", artifactId = "bctls-jdk18on"),
```

Comment:
Is this dependency required for BC JSSE FIPS? I see only next two in https://quarkus.io/guides/security-customization#bouncy-castle-jsse-fips

Reply:
I think this is base dependency for BC and all others are somehow based on it. I don't know if it needs to be there or if it is just transitive dependency or something.

Reply:
I'll try it and report back.

Reply:
I have tried it without it and it works. I still seem to remember they are somehow related, but either I am wrong or I don't want to spend time with it because BC project uses gradle and I can't find an easy way to make link. Good catch